### DISCUSSION TITLE
Dockge does not have access to Stack

### DISCUSSION THREAD

**Discussion by kbgvirus** [louislam/dockge#352] [Feature Request]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

My Stacks would show up, if I click on a stack the message `Dockge does not have access to Stack` would pop up

*** Fixed this issue by renaming all stack folders to lowercase***

### 👟 Reproduction steps

Create stack folder with an uppercase starting letter

### 👀 Expected behavior

Dockge would still work as described

### 😓 Actual Behavior

`Dockge does not have access to Stack`

### Dockge Version

Version: 1.4.1

### 💻 Operating System and Arch

Ubuntu 22.04

### 🌐 Browser

Microsoft Edge 120.0.2210.121

### 🐋 Docker Version

Docker version 24.0.7, build afdd53b

**Comment by louislam** [comment]
Yes, it is a current limitation, it supports lowercase only.

**Reply by kbgvirus** [reply]
Thanks for the clarification
